packages/hooks/useIpfs.ts

```diff
@@ -33,10 +36,20 @@ export const useIpfs = () => {
     async ({
       file,
       pinataJWTKey,
+      mode,
     }: PinataFileProps): Promise<string | undefined> => {
       try {
         const formData = new FormData();
-        formData.append("file", file.file);
+        if (mode === "mini") {
+          //@ts-expect-error: description instead of adding file when adding url in formdata file upload to pinata
```

Comment:
I don't understand the comment

Reply:
As I state in above comment, to create formData for app side, I did it in that way.